fdbclient/Schemas.cpp

```diff
@@ -173,6 +173,7 @@ const KeyRef JSONSchemas::statusSchema = R"statusSchema(
                      "throttled_commits":12341234,
                      "write_stall_microseconds":12341234
                   },
+                  "rocksdb_version":"9.7.3",
```

Comment:
I think this is also needed in `documentation/sphinx/source/mr-status-json-schema.rst.inc` file.